recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -84,6 +85,7 @@ if defined CI (
 
 :: Make paths like C:\hostedtoolcache\windows\Ruby\2.5.7\x64\bin garbage
 set "PATH=%PATH:ostedtoolcache=%"
+set "PATH=%PATH:xternals\git\mingw=%"
```

Comment:
is this the correct spelling?

Reply:
It is a substring. It has some numbers in it, so I make it garbage like the comment says